website/docs/r/advanced_cluster.html.markdown

```diff
@@ -537,11 +537,11 @@ replication_specs {
 
 ### electable_specs
 
+* `instance_size` - (Required) Hardware specification for the instance sizes in this region. Each instance size has a default storage and memory capacity. The instance size you select applies to all the data-bearing hosts in your instance size.
```

Comment:
Do we have the same with the cluster resource?

Reply:
no, cluster resource doesn't have this fields, checked already